src/app/projects/projects.html

```diff
@@ -1,5 +1,82 @@
 <section layout="column" flex>
-  <navbar projects="$projects.projects" user="gapi.user"></navbar>
+  <navbar projects="projects.map" user="gapi.user"></navbar>
+  <div ng-if="ACTIVE_STATE.name === 'projects'">
```

Comment:
Instead of an `ng-if` is there some way we could specify a different state or view for this?